pkg/adapter/adapter.go

```diff
@@ -114,14 +114,12 @@ func (adp *LocalAdapter) Convert(ctx context.Context, source string) error {
 		return err
 	}
 	adp.content.GcMutex.RLock()
+	defer adp.content.GcMutex.RUnlock()
 	if _, err = adp.cvt.Convert(ctx, source, target, cacheRef); err != nil {
 		adp.content.GcMutex.RUnlock()
```

Comment:
This line should be removed?

Reply:
In my point, we should unlock the `GcMutex` after converting.

Reply:
It seems `defer adp.content.GcMutex.RUnlock()` will do.